internal/cli/resume.go

```diff
@@ -0,0 +1,195 @@
+  flightctl resume devices --selector env=staging,tier!=api`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := o.Complete(cmd, args); err != nil {
+				return err
+			}
+			if err := o.Validate(args); err != nil {
+				return err
+			}
+			ctx, cancel := o.WithTimeout(cmd.Context())
+			defer cancel()
+			return o.Run(ctx, args)
+		},
+		SilenceUsage: true,
+	}
+	o.Bind(cmd.Flags())
+	return cmd
+}
+
+func (o *ResumeOptions) Bind(fs *pflag.FlagSet) {
+	o.GlobalOptions.Bind(fs)
+	fs.StringVarP(&o.LabelSelector, "selector", "l", o.LabelSelector, "Selector (label query) to filter on, supporting operators like '=', '!=', and 'in' (e.g., -l='key1=value1,key2!=value2,key3 in (value3, value4)').")
+}
+
+func (o *ResumeOptions) Complete(cmd *cobra.Command, args []string) error {
+	if err := o.GlobalOptions.Complete(cmd, args); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (o *ResumeOptions) Validate(args []string) error {
+	if err := o.GlobalOptions.Validate(args); err != nil {
+		return err
+	}
+
+	arg := args[0]
+
+	// Handle bulk resume case
+	if arg == "devices" {
+		if o.LabelSelector == "" {
+			return fmt.Errorf("label selector is required when resuming multiple devices. Use --selector or -l flag")
+		}
+		return nil
+	}
+
+	// Handle single device case
+	kind, name, err := parseAndValidateKindName(arg)
+	if err != nil {
+		return err
+	}
+
+	if kind != DeviceKind {
+		return fmt.Errorf("kind must be Device")
+	}
+
+	if len(name) == 0 {
+		return fmt.Errorf("specify a specific device to resume")
+	}
+
+	if o.LabelSelector != "" {
+		return fmt.Errorf("label selector cannot be used when resuming a specific device")
+	}
+
+	return nil
+}
+
+func (o *ResumeOptions) Run(ctx context.Context, args []string) error {
+	c, err := o.BuildClient()
+	if err != nil {
+		return fmt.Errorf("creating client: %w", err)
+	}
+
+	arg := args[0]
+
+	// Handle bulk resume case
+	if arg == "devices" {
+		return o.runBulkResume(ctx, c)
+	}
+
+	// Handle single device case
+	_, name, err := parseAndValidateKindName(arg)
+	if err != nil {
+		return err
+	}
+
+	return o.runSingleResume(ctx, c, name)
+}
+
+func (o *ResumeOptions) runSingleResume(ctx context.Context, c *apiclient.ClientWithResponses, name string) error {
+	// Use the bulk resume API with a field selector to target the specific device
+	fieldSelector := fmt.Sprintf("metadata.name=%s", name)
+	request := api.DeviceResumeRequest{
+		FieldSelector: &fieldSelector,
+	}
+
+	response, err := c.ResumeDevicesWithResponse(ctx, request)
+	if err != nil {
+		return fmt.Errorf("resuming device %s: %w", name, err)
+	}
+
+	if response.HTTPResponse != nil {
+		switch response.HTTPResponse.StatusCode {
+		case http.StatusOK:
+			if response.JSON200 != nil {
+				count := *response.JSON200
+				if count > 0 {
+					fmt.Printf("Device resumed successfully: %s\n", name)
+				} else {
+					return fmt.Errorf("device not found or not in conflictPaused state: %s", name)
```

Comment:
The call should be idempotent, like delete.  If it's not in conflictPaused state then we should return 200.

You should also match the success message to delete:
"Resume request for Device \"%s\" completed"

Reply:
done